src/rp2_common/pico_bootrom/include/pico/bootrom.h

```diff
@@ -1071,6 +1071,28 @@ static inline int rom_get_last_boot_type(void) {
  */
 int rom_add_flash_runtime_partition(uint32_t start_offset, uint32_t size, uint32_t permissions);
 
+/*! \brief  Pick A/B partition for a separate partition
+ * \ingroup pico_bootrom
+ *
+ * This will call `rom_pick_ab_partition` with the current `flash_update_boot_window_base`, while performing extra checks to prevent disrupting a main image TBYB.
```

Comment:
Thanks for adding the extra wording. Should the user always prefer using `rom_pick_ab_update_partition` over `rom_pick_ab_partition`, or do they serve different purposes?
Probably worth adding a `\see rom_pick_ab_partition` here? (and vice-versa in the other function's doxygen)

Reply:
I think that if you’re calling it in a TBYB binary before calling `explicit_buy` you should prefer this - for anything else the normal bootrom function is fine

Where does the \see go in the doxygen - is it just below \ingroup?

Reply:
> Where does the \see go in the doxygen - is it just below \ingroup?

I think it can go pretty much anywhere? https://github.com/raspberrypi/pico-sdk/blob/develop/src/rp2_common/hardware_irq/include/hardware/irq.h has lots of examples.

Reply:
Thanks - added